components/widgets/menu/menu_items/submenu_item.tsx

```diff
@@ -44,6 +46,7 @@ type Props = {
     show?: boolean;
     direction?: 'left' | 'right';
     openUp?: boolean;
+    categorySortingMenu?: boolean;
```

Comment:
We shouldn't be adding anything sidebar-specific to this since it's a common component.

Reply:
There are many new style changes it's impossible to override without breaking anything unless there is an additional prop. We can take this into consideration when we do the Menu common component, which I advocate to be soon :P

Reply:
What's different with the sorting menu? If it's the checkboxes and selected item, I think we could have a more general name for these styles and props.